iroha/src/sumeragi.rs

```diff
@@ -2002,6 +2009,9 @@ mod tests {
             config.private_key = keys[i].private_key.clone();
             config.public_key = ids[i].public_key.clone();
             config.sumeragi_configuration.trusted_peers(ids.clone());
+            if i == 0 {
+                config.genesis_configuration.genesis_block_path = Some(GENESIS_PATH.to_string());
+            }
```

Comment:
You don't have to submit genesis block in sumeragi tests, just put some mock data into WSV as it is created in these tests.